src/views/textview/text-view.js

```diff
@@ -338,7 +338,6 @@ export class TextView extends LitElement {
                   id="text-view-middle"
                   .score="${this.score}"
                   .fileName="${this.fileName}"
-                  .limitCollection="${this.limitCollection}"
```

Comment:
Is this correct? This is in the middle column, not the right column. Maybe it is not needed there ...

Reply:
You are right, its obsolete. Didn't notice...

Reply:
ah wait! notice the -, this line is supposed to be deletd. that's right! that should happen!